blogpost_14.html

```diff
@@ -0,0 +1,341 @@
+                  <p>Plans are underway to add Wayland support to Xfwm4 while preserving its existing X11 functionality. However, such a restructurization will be a major effort and we cannot tell yet when/if it will be done, so please don't hold your breath waiting for it.</p>
+
+                  <p>For few components and multiple features, it was not possible to port them to Wayland, partly because there are no standardized Wayland protocols available yet to provide the required services:</p>
+                  <ul>
+                    <li>Workspace support is missing (ext-workspace protocol <a href="https://gitlab.freedesktop.org/wayland/wayland-protocols/-/merge_requests/40">was just merged</a>).</li>
+                    <li>Systray icons will be missing for multiple applications (required to use StatusNotifier instead of GtkStatusIcon)</li>
+                    <li>Xfce4-settings: Keyboard and mouse settings so far are internal to the Wayland compositor and as such the according Xfce dialogs won't be available on Wayland yet.</li>
+                    <li>Power related keyboard handling (brightness keys, suspend, etc.) is internal to the Wayland compositor and as such cannot be supported yet by Xfce components.</li>
+                    <li>Wayland does not specify a native protocol for taking screenshots. However, xfce4-screenshooter already supports screenshots via the wlroots 'screencopy' protocol (entire screen). Screenshots of a rectangle selection or the active window are not yet supported via Wayland.</li>
+                  </ul>
+                  <p>Some other Xfce components don't run at all on Wayland:</p>
+                  <ul>
+                    <li>Xfwm4</li>
+                    <li>Xfdashboard</li>
+                    <li>Xfce4-screensaver (porting to Wayland <a href="https://gitlab.xfce.org/apps/xfce4-screensaver/-/merge_requests/28">completed but not merged</a>, as it requires the experimental <a href="https://gitlab.xfce.org/kelnos/libwlembed">libwlembed library</a>)</li>
+                    <li>Xfce4-windowck-plugin (needs to be <a href="https://gitlab.xfce.org/panel-plugins/xfce4-windowck-plugin/-/issues/37">ported to libxfce4windowing</a>)</li>
+                    <li>Xfce4-xkb-plugin (keyboard handling is internal to the Wayland compositor)</li>
+                  </ul>
+
+                  <p>More detailed info on the current Wayland status can be found on our <a href="https://wiki.xfce.org/releng/wayland_roadmap">Wayland roadmap.</a></p>
+                  <p class="note">@Gael, @Brian: In this section probably many things are not correct or missing. Please just open a MR or send me diffs to modify this section according to your needs. (Should we add any pictures?)
```

Comment:
I think it's fine as-is; the bits you wrote sound good.  Yes, we should update the roadmap and direct people there.